src/index.js

```diff
@@ -1,98 +1,71 @@
-import { deprecateComponents } from './utils/deprecate'
+import { deprecateComponent } from './utils/deprecate'
 
+// ----------------------------------------
 // Addons
-import Confirm from './addons/Confirm/Confirm'
-import Textarea from './addons/Textarea/Textarea'
+// ----------------------------------------
+export Confirm from './addons/Confirm/Confirm'
+export Textarea from './addons/Textarea/Textarea'
 
+// ----------------------------------------
 // Collections
-import Form from './collections/Form/Form'
-import FormField from './collections/Form/FormField'
-import FormFields from './collections/Form/FormFields'
-import Grid from './collections/Grid/Grid'
-import GridColumn from './collections/Grid/GridColumn'
-import GridRow from './collections/Grid/GridRow'
-import Menu from './collections/Menu/Menu'
-import MenuItem from './collections/Menu/MenuItem'
-import Message from './collections/Message/Message'
-import Table from './collections/Table/Table'
-import TableColumn from './collections/Table/TableColumn'
+// ----------------------------------------
 
-// Elements
-import Button from './elements/Button/Button'
-import Buttons from './elements/Button/Buttons'
-import Container from './elements/Container/Container'
-import Divider from './elements/Divider/Divider'
-import Header from './elements/Header/Header'
-import Icon from './elements/Icon/Icon'
-import Image from './elements/Image/Image'
-import Input from './elements/Input/Input'
-import List from './elements/List/List'
-import ListItem from './elements/List/ListItem'
-import Segment from './elements/Segment/Segment'
-import Segments from './elements/Segment/Segments'
+import _Form from './collections/Form/Form'
+export { _Form as Form }
+export const Field = deprecateComponent('Field', 'Use "Form.Field" instead.', _Form.Field)
+export const Fields = deprecateComponent('Fields', 'Use "Form.Fields" instead.', _Form.Fields)
```

Comment:
New deprecated component pattern.  We export each deprecated component separately.